vars/main.yml

```diff
@@ -39,6 +39,11 @@ jenkins_plugins_dir: "{{ jenkins_home }}/plugins"
 # Directory for secret files (credentials, etc)
 jenkins_secrets_dir: "{{ jenkins_root }}/secrets"
 
+# Name of the Jenkins service, as listed by the actual service manager. This is only used
+# when inspecting the output of the host's services, not when creating, starting, or
+# stopping the actual service.
```

Comment:
We could technically pass `jenkins.service` to `ansible.builtin.systemd` (and to `ansible.builtin.service`, which "acts as a proxy to the underlying service manager module").

Reply:
True, but I'd rather not. Let's at least make this comment shorter. Fixed in 809d01f.